lms/templates/seq_module.html

```diff
@@ -23,6 +24,13 @@
     </button>
     <nav class="sequence-list-wrapper" aria-label="${_('Sequence')}">
       <ol id="sequence-list" role="tablist">
+        % if gated_content['gated']:
+        <li>
+          <button class="active nav-item tab" title="${_('Content Locked')}" role="tab" tabindex="-1" aria-selected="false" aria-expanded="false" aria-controls="seq_content" disabled>
```

Comment:
this button needs an ```id="tab_0"``` and the aria-selected and aria-expanded values should both be true.

Reply:
done